pkg/odo/cli/pipelines/bootstrap_test.go

```diff
@@ -1,12 +1,42 @@
 package pipelines
 
 import (
+	"bytes"
 	"regexp"
 	"testing"
 
 	"github.com/spf13/cobra"
 )
 
+// TODO: set up for complete BootstrapOptions instead of just prefix.
+func TestCompleteBootstrapOptions(t *testing.T) {
+	completeTests := []struct {
+		name       string
+		prefix     string
+		wantPrefix string
+	}{
+		{"no prefix", "", ""},
+		{"prefix with hyphen", "test-", "test-"},
+		{"prefix without hyphen", "test", "test-"},
+	}
+
+	for _, tt := range completeTests {
+		o := BootstrapOptions{prefix: tt.prefix}
+
+		err := o.Complete("test", &cobra.Command{}, []string{"test", "test/repo"})
+
+		if err != nil {
+			t.Fatalf("Complete() %#v failed: ", err)
```

Comment:
Please chnage Fatalf to Errorf so it will continue the test to run the next test case.